qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -335,6 +335,11 @@ def _follow_selected_cb(self, js_elem, tab=False):
         """
         if js_elem is None:
             return
+        if js_elem == "focused":
+            # we had a focused element, not a selected one. Just send <enter>
+            self._tab.key_press(Qt.Key_Enter)
```

Comment:
We should probably use ctrl-enter if `tab` is given, just like in `follow_selected`?